### PR TITLE
reef: debian: recursively adjust permissions of /var/lib/ceph/crash

### PR DESCRIPTION
**Pull request by k0ste**
backport tracker: https://tracker.ceph.com/issues/66860

---

backport of https://github.com/ceph/ceph/pull/55917
parent tracker: https://tracker.ceph.com/issues/64548

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/main/src/script/ceph-backport.sh

### CODE REVIEW

**Comment by frittentheke** [comment]
@k0ste @ljflores could the CI errors to render the docs be fixed somehow, so this backport could move forward?

**Comment by k0ste** [comment]
> @k0ste @ljflores could the CI errors to render the docs be fixed somehow, so this backport could move forward?

Rebased, may be this help

**Review by tchaikov — APPROVED** [review]

**Comment by frittentheke** [comment]
@tchaikov @k0ste  Will this be merged to Reef then?

**Comment by k0ste** [comment]
> @tchaikov @k0ste Will this be merged to Reef then?

This decision can make a developer with merge rights

**Comment by tchaikov** [comment]
will merge this PR once https://shaman.ceph.com/builds/ceph/wip-kefu-reef-pr-58458/6fc96426d0d61aad7480b65ef37e8be1b0f8e7a7/ is green.

**Pull request merged**
